drivers/si70xx/si70xx.c

```diff
@@ -18,22 +18,25 @@
  * @}
  */
 
-
+#include <string.h>
 #include "xtimer.h"
 
 #include "si70xx.h"
 
+#define SI70XX_I2C     (dev->params.i2c_dev)
```

Comment:
these macros are (always) defined in the respective c files of most drivers to serve as shortcuts, the preprocessor replaces them directly. There will be no name clash between different drivers, as long as you do not define BUS or ADDR in any header file included by the driver - that is this c file. IMHO its safe to use BUS and ADDR, but I was only suggesting (not demanding) to use these common short names to be consistent with most (or nearly all) other drivers.